balboa-http/README.md

```diff
@@ -45,3 +45,6 @@ sbt balboa-http/test
 sbt balboa-http/run "-Dbalboa.config=balboa-agent/src/main/resources/config/config.properties"
 sbt balboa-http/it:test
 
+### Load Tests
+sbt gatling-it:test
```

Comment:
Add that to the docs. It's a pretty normal scenario for someone to show up here and want to run this against some arbitrary endpoint. As a user, I would just want to see an example to copy and paste, I wouldn't want to dive into the code to figure out where the setting is used, where it came from and how to set it. If `sbt gatling-it:test --help` could be made to do the right thing, that would be best, but a decent example in the README is second best.